fuel-tx/src/transaction/types/input/coin.rs

```diff
@@ -95,6 +131,10 @@ impl CoinSpecification for Full {
 #[derive(Default, Derivative, Clone, PartialEq, Eq, Hash)]
 #[derivative(Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "da-compression",
+    derive(fuel_compression::Compress, fuel_compression::Decompress)
+)]
 #[derive(fuel_types::canonical::Deserialize, fuel_types::canonical::Serialize)]
 pub struct Coin<Specification>
```

Comment:
Fields like `owner`, `amount`, `asset_id`, `tx_pointer` can be skipped since this information should be available from the past

Reply:
Removed in d23d6cd3c5d78ad0115aab5df39627902cfab374